doc/Caliptra.md

```diff
@@ -1152,9 +1172,9 @@ To ensure that the security claims of Caliptra are achieved, specific fuse prote
 
 All fuse based cryptographic keying material and seeds (for example, UDS Seed) shall be generated (on-chip or off-chip) per requirements described in [Reference 8](#ref-8).
 
-SoC shall support in-field programmable fusing. [Fuse Map](#fuse-map) shows which fuses are expected to be in-field programmable. SoCs shall implement authorization for in-field programmable fusing to mitigate denial-of-service attacks. Authorization design is outside the scope of this specification.
+SoC shall support in-field programmable fusing. [Fuse Map](#fuse-map) shows which fuses are expected to be in-field programmable. SoCs shall implement authorization for in-field programmable fusing to mitigate denial-of-service attacks. Authorization design is outside the scope of this specification. In Subsystem mode, SOC may use MCU RT FW for these actions.
```

Comment:
why limit to Subsystem?

Reply:
all this part is saying is that the SOC has a choice to MCU RT FW for doing in-field programming. Not limiting to subsystem. Let me know if the statement is reading wrong somehow

Reply:
Yes I think this sentence is unnecessary.